doc/api/child_process.md

```diff
@@ -1368,6 +1375,34 @@ It is also recommended that any `'message'` handlers in the child process
 verify that `socket` exists, as the connection may have been closed during the
 time it takes to send the connection to the child.
 
+###subprocess.signalCode:
+
+* {integer}
+
+`subprocess.signalCode` property indicates the signal number received by
+the child process if any, from the parent node prcess, else null.
```

Comment:
```suggestion
the child process if any, else null.
```

(The property is not restricted to signals delivered from the parent process.)

Reply:
@addaleax, ditto.